WW3_PreProc/ww3_strt.inp

```diff
@@ -1,13 +1,25 @@
 $ WAVEWATCH III Initial conditions input file
 $ -------------------------------------------
+$ type of initial field ITYPE .
+$
+    3
+$
 $ ITYPE = 1, Gaussian in frequency and space, cos type in direction.
-   1
 $  fp        sip    thm   ncos xm     six    ym    siy     hmax
-   0.040469  0.0001 225.2 200  29.14 -0.5000 83.44  0.5000   1.0
+$  0.040469  0.0001 225.2 200  29.14 -0.5000 83.44  0.5000   1.0
 $  0.040469  0.0001  90.0 200  85.00 -0.5000 85.00  0.5000   1.0
 $ BGR: Replace with line below to test along seam-propagation.
 $  0.040469  0.0001   0.0 200  85.00 -0.5000 85.00  0.5000   1.0
 $ ITYPE=2, JONSWAP with Gaussian in space
 $  2
 $  alpha  fp       thm  gamma siga sigb xm     six    ym   siy   
 $  0.0081 0.040469 180. 1.0   0.07 0.09 210.0 -0.5000 83.0 0.5000
+$ ITYPE = 3 ---------------------------------------------------------- $
+$ Fetch-limited JONSWAP
+$ - No additional data, the local spectrum is calculated using the
+$   local wind speed and direction, using the spatial grid size as
+$   fetch, and assuring that the spectrum is within the discrete
+$   frequency range.
+$ -------------------------------------------------------------------- $
+$ End of input file                                                    $
+$ -------------------------------------------------------------------- $
```

Comment:
I haven't checked these. Could you please double check that the changes are correct.

Reply:
These changes are correct